.github/workflows/release.yml

```diff
@@ -145,16 +145,19 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Setup
+        if: ${{ github.repository_owner == 'ngxs' }}
```

Comment:
Should we remove braces?

> When you use expressions in an if conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression.